bundledata_test.go

```diff
@@ -344,6 +344,8 @@ applications:
             "gui-y": -15
         resources:
             "": 42
+            "foo":
+               "not": int
```

Comment:
What's "not" mean?

Reply:
nothing. it could be any value at all. i just meant it to look like "not int"..